draft-ietf-rtcweb-jsep.xml

```diff
@@ -4065,11 +4065,11 @@ BobJS->BobUA:       setRemoteDescription with |offer-B1|
 BobUA->BobJS:       ontrack with audio track from Alice
 
 //                  candidates are sent to Bob
-AliceUA->AliceJS:   onicecandidate event with |offer-B1-candidate-1| (host)
```

Comment:
I could live with just dropping the parentheticals (and preserving 'event'), but I could go either way.